modules/nf-core/sourmash/taxannotate/meta.yml

```diff
@@ -0,0 +1,59 @@
+name: "sourmash_taxannotate"
+description: Annotate list of metagenome members (based on sourmash signature matches) with taxonomic information.
+keywords:
+  - FracMinHash sketch
+  - signature
+  - kmer
+  - containment
+  - sourmash
+  - genomics
+  - metagenomics
+  - taxonomic classification
+  - taxonomic profiling
+tools:
+  - "sourmash":
+      description: Compute and compare FracMinHash signatures for DNA data sets.
+      homepage: https://sourmash.readthedocs.io/
+      documentation: https://sourmash.readthedocs.io/
+      tool_dev_url: https://github.com/sourmash-bio/sourmash
+      doi: "10.21105/joss.00027"
+      licence: ["BSD-3-clause"]
+
+## Description of all of the variables used as input
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - gather_results:
+      type: file
+      description: |
+        Table with signatures classified as belonging to any of the genomes
```

Comment:
```suggestion
        Mandatory table with signatures classified as belonging to any of the genomes
```

Reply:
Note updated docmentation guidelines [^1], but otherwise this documentation was already really good :+1: :muscle: 

[^1]: https://nf-co.re/docs/contributing/modules#documentation